netrd/distance/graph_diffusion_distance.py

```diff
@@ -0,0 +1,156 @@
+"""
+gdd.py
```

Comment:
nitpick: either call the file `gdd.py` or change this to the actual name of the file.

nitpick: since this file is inside the `distance/` folder, I'd rename the file to `graph_diffusion.py`-

Reply:
Yeah, it was originally called `gdd.py` and I renamed it to `graph_diffusion_distance.py`. Standardized it to `graph_diffusion.py`.